Gemfile

```diff
@@ -32,8 +32,10 @@ group :development, :test do
   gem "capybara"
   gem "factory_bot_rails"
   gem "govuk-lint", "~> 3"
+  gem "govuk_test"
   gem 'pry'
   gem "rspec-rails", "~> 3"
+  gem "selenium-webdriver"
```

Comment:
I think the `govuk_test` gem requires `selenium-webdriver` so I'm wondering if having this entry in the Gemfile is necessary/useful?

Reply:
Good spot, removed with no ill effect.